generate-rules.sh

```diff
@@ -1,25 +1,47 @@
 #!/usr/bin/env bash
+# dependencies: git, jq, tr (coreutils), awk (gawk), sed (gnused)
 
+for c in git jq tr awk; do
```

Comment:
```suggestion
for c in git jq tr awk sed; do
```
since `sed` is now used